SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -328,10 +331,13 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	Sender->Init(this, &TimerManager);
 	Receiver->Init(this, &TimerManager);
 	GlobalStateManager->Init(this, &TimerManager);
-	VirtualWorkerTranslator->Init(this);
-	// TODO(zoning): This currently hard codes the desired number of virtual workers. This should be retrieved
-	// from the configuration.
-	VirtualWorkerTranslator->SetDesiredVirtualWorkerCount(2);
+	if (GetDefault<USpatialGDKSettings>()->bEnableUnrealLoadBalancer)
+	{
+		VirtualWorkerTranslator->Init(this);
```

Comment:
I checked all other referenced to the translator in the code base, and they're all behind null checks.

Reply:
Could this line be moved above, right after the `VirtualWorkerTranslator = MakeUnique<SpatialVirtualWorkerTranslator>();`?

Reply:
Perhaps, but I don't want to change anything I don't have to, and perhaps there's subtle order dependencies with these objects. This code has been re-written in master already.